spec/support/jasmine-browser.json

```
@@ -0,0 +1,25 @@
+{
+  "srcDir": "spec/dummy/public/assets",
+  "srcFiles": [
+    "govuk_publishing_components/dependencies-*.js",
+    "govuk_publishing_components/all_components-*.js",
+    "govuk_publishing_components/modules-*.js",
+    "govuk_publishing_components/analytics-*.js",
+    "component_guide/accessibility-test-*.js",
+    "component_guide/filter-components-*.js"
+  ],
```

I've noticed that the CSS file isn't included with this config, but it was previously included when using the gem.

This may not be a problem, so long as the Jasmine tests only cover functionality and don't cover styling such as `element.style.backgroundColor` etc.

However, for the sake of being comprehensive and making things behave as expected, it'd probably be worth adding it in here:

```suggestion
  ],
  "cssFiles": [
    "application-*.css",
    "component_guide/application-*.css"
  ],
```

---

Sure, I've added it in for equivalence. Somewhat mixed feelings on whether it should be in but could see it needed for a component considering DOM visual state.

---

If nothing else, I think it just reduces the "WTF factor" if anything does come to depend upon CSS. I don't think the `cssFiles` option was as well documented, so this avoids somebody trying to scramble around to find it in the future.

I expect the [visual regression tests](https://github.com/alphagov/govuk_publishing_components/blob/master/.github/workflows/visual-regression-tests.yml) are probably the right way to test most of our CSS though.